framework/core/command_buffer.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2019-2020, Arm Limited and Contributors
+/* Copyright (c) 2019-2021, Arm Limited and Contributors
  *
  * SPDX-License-Identifier: Apache-2.0
  *
```

Comment:
line 81: move constructors should be marked noexcept

line 161: End of the line has an extra ';'.  The macro LOGE handles the ';'